product_docs/docs/postgres_for_kubernetes/1/security.mdx

```diff
@@ -72,9 +71,12 @@ following tools:
     and reports findings via the GitHub interface.
 
 !!! Important
-    All operand images are automatically rebuilt daily by our pipelines to
-    incorporate security updates at the base image and package level, providing
-    **patch-level updates** for the container images distributed to the community.
+All operand images are automatically rebuilt daily by our pipelines to incorporate security updates at the base image and package level, providing **patch-level updates** for the container images distributed to EDB download sites.
+!!!
+
+!!! Warning
+Running outdated images can expose your environment to security risks and performance issues. We highly recommend that you update to the latest image version and keep your images up to date. This will ensure that you take advantage of the latest updates and patches available.
```

Comment:
Maybe add a link to [rolling_update.mdx](https://www.enterprisedb.com/docs/postgres_for_kubernetes/latest/rolling_update/) here?